### PR TITLE
Removes `standalone` from diagnostics README

### PR DESCRIPTION
**Pull request by jpmcb**
## What this PR does / why we need it
Removes stale references to `standalone` in the Diagnostics plugin 

## Details for the Release Notes (PLEASE PROVIDE)
```release-note
Removed stale references to standalone-cluster in the diagnostics plugin
```

## Which issue(s) this PR fixes
Related to: #3382

### CODE REVIEW

File: cli/cmd/plugin/diagnostics/README.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ Flags:
       --management-cluster-name string         The name of the management cluster (required)
       --management-cluster-skip                If true, skips management cluster diagnostics
       --output-dir string                      Output directory for collected bundle (default "./")
-      --standalone-cluster-context string      The context name of the standalone cluster
-      --standalone-cluster-kubeconfig string   The standalone cluster config file (required) (default "${HOME}/.kube/config")
-      --standalone-cluster-name string         The name for the standalone cluster (required)
+      --unmanaged-cluster-context string      The context name of the standalone cluster
```
Comment (stmcginnis) [inline cli/cmd/plugin/diagnostics/README.md:74]: Should we update the flag descriptions too?

Comment (stmcginnis) [reply cli/cmd/plugin/diagnostics/README.md:74]: Also looks like there needs to be one more space before the description so it aligns with the rest of them.

Comment (jpmcb) [reply cli/cmd/plugin/diagnostics/README.md:74]: Whomp whomp, `%s/` has failed me!!! Thanks for the thoughtful review

Comment (jpmcb) [reply cli/cmd/plugin/diagnostics/README.md]: Updated - built the binary and replaced the whole section with what was generated by `--help`. So should accurately reflect spacing and what the command helps are

File: cli/cmd/plugin/diagnostics/README.md [hunk 2]
```diff
@@ -71,13 +71,14 @@ Flags:
       --management-cluster-name string         The name of the management cluster (required)
       --management-cluster-skip                If true, skips management cluster diagnostics
       --output-dir string                      Output directory for collected bundle (default "./")
-      --standalone-cluster-context string      The context name of the standalone cluster
-      --standalone-cluster-kubeconfig string   The standalone cluster config file (required) (default "${HOME}/.kube/config")
-      --standalone-cluster-name string         The name for the standalone cluster (required)
-      --work-dir string                        Working directory for collected data (default "${HOME}/.config/tanzu/diagnostics")
+      --unmanaged-cluster-context string       The context name of the unmanaged cluster
+      --unmanaged-cluster-kubeconfig string    The unmanaged cluster config file (required) (default "/home/jmcb/.kube/config")
```
Comment (stmcginnis) [inline cli/cmd/plugin/diagnostics/README.md:75]: Sorry, one more - can we keep the `${HOME}` placeholder for the paths?

Comment (jpmcb) [reply cli/cmd/plugin/diagnostics/README.md]: 🙈

**Review by stmcginnis — APPROVED** [review]
Looks great! Thanks for the updates.

**Pull request merged**